Telemetry/PingCentre.swift

```diff
@@ -10,6 +10,7 @@ import Deferred
 // MARK: Ping Centre Client
 public protocol PingCentreClient {
     @discardableResult func sendPing(_ data: [String: Any], validate: Bool) -> Success
+    @discardableResult func sendBatch(_ data: [[String: Any]], validate: Bool) -> Success
```

Comment:
Ohhh cool! I was wondering about this the other day as to whether or not you can specify that a return value can be optionally discarded. Nice! I might use this in the Telemetry library :-)